plugins/advisors/black-duck/src/main/kotlin/BlackDuck.kt

```diff
@@ -199,12 +199,11 @@ class BlackDuck(
 
 internal fun VulnerabilityView.toOrtVulnerability(): Vulnerability {
     val referenceUris = setOf(meta.href.uri(), *meta.links.map { it.href.uri() }.toTypedArray())
+    val cvssVector = cvss3?.vector ?: cvss2?.vector
+    // Only CVSS version 2 vectors do not contain the "CVSS:" label and version prefix
```

Comment:
While at it, mind adding a dot at the end of the sentence here to align with the same code in `OssIndex.kt`?